public/plugin.ts

```diff
@@ -64,6 +64,10 @@ export class WazuhPlugin implements Plugin<WazuhSetup, WazuhStart, WazuhSetupPlu
   }
 
   public async start(core: CoreStart, plugins: AppPluginStartDependencies): Promise<WazuhStart> {
+    // hide security alert 
+    if(plugins.security_oss)
+      plugins.security_oss.insecureCluster.hideAlert(true);
```

Comment:
It would be `plugins.securityOss` instead `plugins.security_oss`. 

As a personal note, I don't like the conditional inline statement without curly braces.